### ISSUE TITLE
Add videos and images for food recipes on the Front page 

### ISSUE THREAD

**Issue by Zedoman** [Anupkjha2601/finalproj#88]
Please assign this issue to me under GSSOC'23.

**Comment by LaxmanSingh07** [comment]
Could you please consider assigning this issue to me under GSSOC'23? I am eager to contribute my knowledge and contribute to the success of this project.

**Comment by Ajay4486** [comment]
Yes I can do this by adding some interesting food recipes gif and images , please assign me this issue .
I would like to contribute this project  under GSSOC'23 .
